shared/actions/kbfs/index.platform.desktop.js

```diff
@@ -169,10 +169,13 @@ function* installFuseSaga(): SagaGenerator<any, any> {
 function findKeybaseUninstallString(): Promise<string> {
   return new Promise((resolve, reject) => {
     const regedit = require('regedit')
-    const uninstallRegPath = 'HKCU\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall'
-    regedit.list(uninstallRegPath).on('data', function(entry) {
-      for (var keyName of entry.data.keys) {
-        regedit.list(uninstallRegPath + '\\' + keyName).on('data', function(entry) {
+    const keybaseRegPath = 'HKCU\\SOFTWARE\\Keybase\\Keybase'
+    regedit.list(keybaseRegPath).on('data', function(entry) {
+      if (entry.data.values.BUNDLEKEY) {
+        const uninstallRegPath =
```

Comment:
Maybe we should check that values isn’t null. Or wrap this stuff in a try catch block maybe